milli/src/search/matches/matching_words.rs

```diff
@@ -225,7 +225,7 @@ fn bytes_to_highlight(source: &str, target: &str) -> usize {
         for (col, char_t) in target.chars().enumerate() {
             let col = col + 1;
             let last_match_row = *last_row.get(&char_t).unwrap_or(&0);
-            let cost = if char_s == char_t { 0 } else { 1 };
+            let cost = usize::from(char_s != char_t);
```

Comment:
Converts a bool to a usize. The resulting value is 0 for false and 1 for true values.

[Examples](https://doc.rust-lang.org/std/primitive.usize.html#examples-87)
```rs
assert_eq!(usize::from(true), 1);
assert_eq!(usize::from(false), 0);
```